Source/JavaScriptCore/assembler/MacroAssemblerRISCV64.h

```diff
@@ -171,6 +171,16 @@ class MacroAssemblerRISCV64 : public AbstractMacroAssembler<Assembler> {
     static constexpr RegisterID framePointerRegister = RISCV64Registers::fp;
     static constexpr RegisterID linkRegister = RISCV64Registers::ra;
 
+    void add8(RegisterID src, RegisterID dest)
+    {
+        add32(src, dest);
+    }
+
+    void add8(TrustedImm32 imm, Address address)
+    {
+        add32(imm, address);
```

Comment:
This is not correct. It is add32. Not add8.

Reply:
Thank you for your review, I will take a closer look later, because you know, due to the limitations of the board(cpu, ram, disk), debugging on it takes a lot of time

Reply:
Maybe something like,

```c++
void add8(TrustedImm32 imm, Address address)
{
    auto temp = temps<Data, Memory>();
    load8(address, temp.memory());
    add32(imm, temp.memory(), temp.data());
    store8(temp.data(), address);
}
```

based on the arm64 implementation, but using the `temp` pattern from elsewhere in the riscv64 header? It's hard to say for sure when it doesn't build :)

Reply:
I think you probably want
```
void add8(TrustedImm32 imm, Address address)
{
    auto temp = temps<Data>();
    load8(address, temp.data());
    add32(imm, temp.data());
    store8(temp.data(), address);
}
```
since `load8` and `store8` are going to use the memory temp.